source/draw/gpu/GpuCanvas.ooc

```
@@ -72,7 +76,7 @@ GpuCanvasYuv420Semiplanar: class extends GpuSurface {
 		this _target y canvas fill()
 		this _target uv canvas fill()
 	}
-	draw: override func ~mesh (image: Image, mesh: GpuMesh) {
+	draw: override func ~mesh (image: GpuImage, mesh: GpuMesh) {
 		yuv := image as GpuYuv420Semiplanar
```

It should at least raise an exception before crashing.